Packs/MISP/ReleaseNotes/2_1_50.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### MISP v3
+
+- Fixed an issue where an indicator with a tag defined in the *Malicious tag IDs* parameter returned a score of `unknown` instead of `malicious`.
```

Comment:
```suggestion
Fixed an issue where an indicator with a tag defined in the *Malicious tag IDs* parameter returned a score of `unknown` instead of `malicious`.
```